user-service/user-service.iml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Please add .gitignore file to project and add THIS file there to delete if from code base